include/deal.II/grid/tria.h

```diff
@@ -1615,6 +1616,22 @@ class Triangulation : public Subscriptor
   virtual MPI_Comm
   get_communicator() const;
 
+  /**
+   * Return partitioner for the global indices of the cells on the active
```

Comment:
I know we have this text also in the derived class, but I believe we typically use the style
```suggestion
   * Return the partitioner for the global indices of the cells on the active
```
See also the documentation of the function `set_mesh_smoothing` below. (On the other hand, the one immediately preceding this does not use the article, so I am a bit split.)